test/dotnet-msbuild.Tests/GivenDotnetCacheInvocation.cs

```diff
@@ -46,7 +47,7 @@ public void MsbuildInvocationIsCorrect(string[] args, string expectedAdditionalA
         [InlineData("--output")]
         public void ItAddsOutputPathToMsBuildInvocation(string optionName)
         {
-            string path = "/some/path";
+            string path = Directory.GetCurrentDirectory();
             var args = ArgsPrefix.Concat(new string[] { optionName, path }).ToArray();
```

Comment:
The parser configuration had a default set to the current directory at one point, which resulted in a failure on Windows. This is no longer an issue and this change can be reverted, but it seems to me like we should either be testing explicitly that both Windows and *nix paths are handled correctly, or we should at least be writing tests in a way that exercises the variations in behavior we expect to see. 

Thoughts?